frontend/src/components/TreeView.js

```diff
@@ -0,0 +1,179 @@
+import React, { useEffect, useState } from 'react'
+import { useNavigate, useLocation } from 'react-router-dom'
+import { Text, Box } from '@chakra-ui/react'
+import getAllChildren from '../utils/getAllChildren'
+
+const TreeView = (props) => {
+    const navigate = useNavigate()
+    const location = useLocation()
+    const { tickets } = props
+    const [tree, setTree] = useState({})
+    const pathname = location.pathname.substring(1).split('/')
+    const ticketType = pathname[0]
+    const ticketID = parseInt(pathname[1])
+
+    const TICKET_TYPES = Object.freeze({
```

Comment:
Rename the DATA_KEYS to TICKET_TYPES and import that instead, just so we only have one  place where it has been defined

Reply:
I changed that in my PR so you can pull my changes and import it from Dashboard.js